appveyor.yml

```diff
@@ -0,0 +1,11 @@
+version: 1.0.{build}
+image: Visual Studio 2013
```

Comment:
We have 3rd party libraries compiled with MSVC 20123. I recently updated all libraries to VS2015. I think we are safe to use VS2015.